src/pages/SignMessage/components/SignMessageSection/components/SignMessageForm.tsx

```diff
@@ -0,0 +1,73 @@
+import { ChangeEvent, useEffect } from 'react';
+import { ErrorMessage, Field, Form, useFormikContext } from 'formik';
+import { CopyButton } from '@multiversx/sdk-dapp/UI/CopyButton';
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { useSignMessageForm } from '../hooks/useSignMessageForm';
+import { SignMessageFormValues } from '../types';
+import styles from '../../../styles.module.scss';
+import { useSignMessageSectionContext } from '../../../context';
```

Comment:
absolute import?

Reply:
No, are related to the SignMessage page module